src/storage.zig

```diff
@@ -268,7 +268,7 @@ pub fn Storage(comptime IO: type) type {
                         return;
                     } else {
                         // We tried to read at (or less than) logical sector granularity and failed.
-                        log.err("latent sector error: offset={}, zeroing sector...", .{
+                        log.warn("latent sector error: offset={}, zeroing sector...", .{
                             read.offset,
```

Comment:
I am unsure about this one --- we expect disk errors to be _very_ rare, so perhaps it's worth keeping it as an `err`?

I guess there's also an angle that it would be _extremely_ interesting to learn how frequently do we get this, so artificially increasing loudness here might be a good thing?

As in, every time someone is paged over "latent sector error", they would feel relieved that they are using TigerBeetle :rofl: